test/Performance/setup_env_unix.sh

```diff
@@ -158,12 +180,12 @@ cp php.ini-production php.ini
 driverName=$(basename $SQLSRV_DRIVER)
 echo "extension=$driverName" >> php.ini
 sudo cp -r $SQLSRV_DRIVER $phpExtDir/$driverName
-sudo chmod a+r $SQLSRV_DRIVER $phpExtDir/$driverName
+sudo chmod a+r $phpExtDir/$driverName
```

Comment:
chmod won't work if SQLSRV_DRIVER is located on a read-only filesystem, and it isn't even needed here once the drivers are copied to the PHP extensions directory. Same for PDO below.